pkgs/servers/sql/postgresql/default.nix

```diff
@@ -6,6 +6,9 @@ let
       , glibc, zlib, readline, openssl, icu, systemd, libossp_uuid
       , pkgconfig, libxml2, tzdata
 
+      # This is important to obtain a version of `libpq` that does not depend on systemd.
+      , enableSystemd ? (lib.versionAtLeast "9.6" && !stdenv.isDarwin)
```

Comment:
This is `[partially applied function] && [boolean]`, so breaks eval. 

(for the reference: the old code used `atLeast` defined as `lib.versionAtLeast version`)

Reply:
Maybe do something like `enableSystemd ? true` and then `assert enableSystemd -> (atLeast "9.6" && !stdenv.isDarwin)`?

Reply:
`version` is already among the args, so just fixing a typo should work; your proposal gives a bad default on macOS.

Reply:
Sorry, fixed.